tldw_Server_API/tests/Agent_Client_Protocol/test_acp_endpoints.py

```
@@ -55,7 +55,7 @@ async def _get_runner_client():
 def test_acp_session_new_success(client_user_only, stub_runner_client):
```

Ruff (0.15.1)

[warning] Unused function argument: `stub_runner_client`

(ARG001)